src/tokenlist.json

```diff
@@ -30,16 +21,7 @@
       "symbol": "CHAT",
       "fee": 100000
     },
-    {
-      "id": "7ajy4-sqaaa-aaaaq-aaaqa-cai",
-      "index_canister": "7olvu-jaaaa-aaaaq-aaara-cai",
-      "logo": "https://raw.githubusercontent.com/infinity-swap/token-lists/main/logos/soc.png",
-      "decimals": 8,
-      "standard": "ICRC1",
-      "name": "SONIC",
-      "symbol": "SOC",
-      "fee": 100000
-    },
+    null,
```

Comment:
Why here is null? This will break the wallet.

Reply:
I have fixed this bug